tremor-cli/tests/integration/tcp-reuseport/before/config.troy

```diff
@@ -10,11 +10,13 @@ flow
       "separate"
     ],
     config = {
-      "url": "127.0.0.1:22224",
       "socket_options": {
         "SO_REUSEPORT": true
       }
-    }
+    },
+    initial_commands = [
+      {"SocketServer": {"Listen": {"address": "tcp://127.0.0.1:22224", "handle": "a"}}}
```

Comment:
leaving this in here for later so we don't forget: we should follow the same convention for naming as every where else `SocketServer` => `socket_server`